apps/starter/src/ensemble/dashboard.yaml

```diff
@@ -264,14 +264,145 @@ View:
         - Text:
             id: myText
             text: Peter Parker
-        - Button:
-            label: Click Me
-            onTap:
-              executeCode: myText.setText("Spiderman")
         - Text:
             text: ${myText.text}
         - Text:
             text: ${getDummyJson.body.total}
+        - Stepper:
+            id: stepper1
+            styles:
+              connectorColor: "#31C48D"
+              connectorHeight: 2
+            steps:
+              - stepLabel: Match
+                inactiveStepWidget:
```

Comment:
@evshi this widget has three states active, inactive and completed. We can also have error state ( not implemented yet ). What are your thoughts for given Yaml syntax ?

Reply:
I think you will want a property that allows you to define the workflow definition like charts does with data. Then you can provide an item-template that will bind the view to each step definition.